### ISSUE TITLE
[MAS3.2.1] [Screen Reader-Live Chat] When the user selected the “Edit ngrok setting” keyboard focus is not lands on the Path to ngrok text field

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1762]
**Actual Result:**
When the user selected the “Edit ngrok setting” keyboard focus is not lands on the Path to ngrok text field
When the user cancels the Emulator setting dialog focus is not lands on the type your message text field.

**Expected Result:**
When the user selected the “Edit ngrok setting” keyboard focus should be lands on the Path to ngrok text field.
When the user cancels the Emulator setting dialog focus should be lands on the type your message text field.

**Note:** When the user selects the edit ngrok setting, narrator focus is moves to hidden and reading as Document.

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened
3. Navigate to the Open a bot and select it 
4.Open bot dialog gets opened and connect the bot 
5.Live chat screen gets opened 
6.Navigate to the “Edit ngrok setting control “using tab key 
7. Observe the issue

**User Impact:**	
Users who rely on keyboard navigation will be affected to navigate the control if keyboard focus is not lands on the expected control keyboard users will be affected.

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Browser: Edge
Application: Bot Framework V4 Emulator

**MAS Reference Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a4bbd37d-ee11-485e-b5c8-0539f0ac94b2}](url)
[MAS3.2.1_Ngrok.zip](https://github.com/microsoft/BotFramework-Emulator/files/3525372/MAS3.2.1_Ngrok.zip)
[MAS3.2.1_Ngrok cancel.zip](https://github.com/microsoft/BotFramework-Emulator/files/3525378/MAS3.2.1_Ngrok.cancel.zip)
[MAS3.2.1_Edit ngrok setting.zip](https://github.com/microsoft/BotFramework-Emulator/files/3525387/MAS3.2.1_Edit.ngrok.setting.zip)

**Comment by Kumar2608** [comment]
Verified in latest Bot emulator version -4.5.2-79718 .Issue got fixed hence we are closing this bug.
